vendor/brave-ios/Ledger/Data/Model/PendingContribution.h

```diff
@@ -15,7 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic) int64_t addedDate;
 @property (nonatomic) double amount;
-@property (nonatomic) int32_t category;
+@property (nonatomic) int32_t type;
```

Comment:
Changing the name of this attribute requires updating:
1. The CoreData entity
2. The `@dynamic category` declaration in PendingContribution.m

Reply:
Fixed